### PR TITLE
fix npm security audit and migrate to yarn

### PR DESCRIPTION
**Pull request by Slashgear**
🎃🎃🎃🎃🎃

`HttpToStatsd` became a spooky scary skeleton with time.
I had to purify it with holy water.

- Node 10 compatibility
- fix `npm audit`
- yarn migration

### CODE REVIEW

**Review by fdubost — APPROVED** [review]

File: package.json [hunk 1]
```diff
@@ -3,23 +3,28 @@
   "version": "0.0.1",
   "description": "HTTP service to log monitoring information to StatsD",
   "main": "app.js",
+  "engines": {
+    "node": ">= 10.11.0",
+    "npm": ">= 6.4.1"
+  },
   "dependencies": {
-    "MD5"        : "~1.1.0",
+    "MD5": "~1.1.0",
```
Comment (valotvince) [inline package.json]: 🇨🇳 🎩

File: package.json [hunk 2]
```diff
@@ -3,23 +3,28 @@
   "version": "0.0.1",
   "description": "HTTP service to log monitoring information to StatsD",
   "main": "app.js",
+  "engines": {
+    "node": ">= 10.11.0",
+    "npm": ">= 6.4.1"
```
Comment (fdubost) [inline package.json]: Maybe add `yarn` here 🦁

**Comment by Osirisxxl** [comment]
👻  Happy Halloween 😄

File: package.json (hunk 2 above)
Comment (Slashgear) [reply package.json]: We don't really have the control of `yarn` version in travis. That's why I prefered not to set this/

**Comment by omansour** [comment]
excellent

File: package.json (hunk 2 above)
Comment (flepretre) [reply package.json]: you can choose you version of yarn in travis, but yes it's ugly ...

**Review by flepretre — APPROVED** [review]

File: package.json (hunk 2 above)
Comment (fdubost) [reply package.json]: But I don't think about travis, I think about user here 😄

**Comment by Slashgear** [comment]
Yeah @fdubost but if I put this config, build will fail. I prefer not
setting it.
Travis is still in yarn 1.6

Le ven. 2 nov. 2018 10:06, Florent Dubost <notifications@github.com> a
écrit :

**Pull request merged**